configuration/pih/htmlforms/ovcIntake.xml

```diff
@@ -0,0 +1,265 @@
+
+<htmlform
+    formUuid="a2f4e400-df59-4456-935f-cd0697ff28a4"
+    formName="OVC Intake"
+    formEncounterType="651d4359-4463-4e52-8fde-e62876f90792"
+    formVersion="1.0">
+
+    <div style="display: none">
+        <encounterDate default="today" />
+    </div>
```

Comment:
Are we allowing the encounter date to be changed at all?

Reply:
Replaced with the block you recommended